app/models/oral_history_content.rb

```diff
@@ -124,15 +131,18 @@ def interviewee_death_place
   end
 
   def interviewee_schools_sorted
-    interviewee_school.sort_by { |hsh| hsh.date }
+    return [] unless interviewee_school.present?
+    interviewee_school.sort_by { |hsh| hsh.date }.compact
   end
```

Comment:
These `sorted` methods should be in a presenter or view-level component of some kind rather than the model. 

Or, we could just skip it entirely -- let the staff entering them enter them in the order they should be, and just respect the order they were entered in?  I bet that's what the microsite did, and may be good enough if we just want to be done with this. But if we're going to have extra "sorted" methods, they shouldn't be muddying up the model, but should be in a presenter or view layer.

Reply:
Got rid of them -- totally unnecessary in retrospect.